src/components/experiments/single-view/ExperimentPageView.tsx

```diff
@@ -156,7 +156,7 @@ export default function ExperimentPageView({
           <Typography className={classes.titleHeader} variant='h2'>
             Experiment:{' '}
             {experiment ? (
-              <Tooltip title={experiment.name}>
+              <Tooltip title={`${experiment.name}`}>
```

Comment:
This will render `null` if the `name` is `null`. Let's use the empty string as a fallback.
```suggestion
              <Tooltip title={experiment.name ?? ''}>
```

Reply:
Ah good find!